recipes/vaex/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "vaex" %}
+{% set version = "1.0.0b3" %}
+{% set sha256 = "14336a98579caeb98afb1bfdcb4c2787208525ea54e355b3c404d247791f3bb1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+   - vaex = vaex.__main__:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7.2
+    - pip
+    - matplotlib
+  run:
+    - python
+    - pyopengl >=3.1.0
+    - numpy >=1.7.2
+    - scipy >=0.1
+    - astropy >=0.3
+    - h5py >=2.2.1
+    - matplotlib >=1.3.1
+    - numexpr
+    - psutil >=1.2.1
+    - aplus
+    - tornado >4.1
+    - futures
+    - future >=0.15.2
+    - pyyaml
+    - cachetools
+    - progressbar2
+    - pandas
+    - pillow
+    - python.app  # [osx]
+
+test:
+  imports:
+    - vaex
+    - vaex.file
+    - vaex.misc
+    - vaex.test
+    - vaex.ui
+    - vaex.ui.icons
+    - vaex.ui.plugin
+
+about:
+  home: http://vaex.astro.rug.nl
+  license: MIT
```

Comment:
The LICENSE file wasn't in the manifest file, I've added it and will include it in the recipe next release.